src/Resources/doc/configuration-reference.md

```diff
@@ -0,0 +1,116 @@
+# Configuration Reference
+
+``` yaml
+eight_points_guzzle:
+    # (de)activate logging/profiler; default: %kernel.debug%
+    logging: true
```

Comment:
`profiling` option is missing here, plus when https://github.com/8p/EightPointsGuzzleBundle/pull/228 lands that should be added as well :smiley:

Reply:
right :slightly_smiling_face: I also plan to add `slow_response_time` after merging your PR.

Reply:
added `profiling` and `slow_response_time`